common/i18n/resources/en.js

```diff
@@ -228,10 +228,10 @@ module.exports = {
     CategoriesValidationJob: 'Categories Validation',
     CollectImportJob: 'Collect Import',
     CollectSurveyReaderJob: 'Collect Survey Reader',
-    EntitiesUniquenessValidationJob: 'Entities Uniqueness Validation',
     NodeDefsImportJob: 'Node Defs Import',
     NodeDefsValidationJob: 'Node Defs Validation',
     RecordCheckJob: 'Record Check',
+    RecordsAndEntitiesUniquenessValidationJob: 'Records and Entities Uniqueness Validation',
```

Comment:
Why having this long name RecordsAndEntitiesUniquenessValidationJob. Could it be like before EntitiesUniquenessValidationJob, and we know that if a root entity is duplicate, it means there's another record with same error?

Reply:
good point, I'll change it.